src/inmanta/parser/plyInmantaParser.py

```diff
@@ -1080,8 +1107,8 @@ def p_constants_collect(p: YaccProduction) -> None:
 
 
 def p_wrapped_kwargs(p: YaccProduction) -> None:
-    "wrapped_kwargs : '*' '*' operand"
-    p[0] = WrappedKwargs(p[3])
+    "wrapped_kwargs : DOUBLE_STAR operand"
+    p[0] = WrappedKwargs(p[2])
```

Comment:
This is a (imo good) breaking change, i.e. we don't support unpacking stuff with spaces anymore e.g. `B(   *  *a)` should we bother mentioning this ?

Reply:
Good point. Another change that I think is intrinsically good, yet technically breaking.